### PR TITLE
Ctrl + Z and (Ctrl + Y and Ctrl + Shift + Z) support

### PR DESCRIPTION
**Pull request by Ascynx**
Every action on the text of a text field can be undone and redone using Ctrl + z and Ctrl + Y.

### CODE REVIEW

**Comment by Ascynx** [comment]
Currently, acting after undoing or redoing an action will append at the end of the stack. I'm still debating with myself wether I want to keep it that way or cut the stack to where the last redo/undo last happened.

File: src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java [hunk 1]
```diff
@@ -1344,6 +1344,7 @@ public boolean checkMatchesSort(String internalname, JsonObject item) {
 	public void updateSearch() {
 		SunTzu.randomizeQuote();
 
+
```
Comment (hannibal002) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1347]: can we remove this extra line?

**Review by hannibal002 — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java (hunk 1 above)
Comment (Ascynx) [reply src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java:1347]: Can do when I'm back home

**Review by NopoTheGamer — APPROVED** [review]

**Review by hannibal002 — APPROVED** [review]

**Review by IRONM00N — APPROVED** [review]
ctrl + shift + z my beloved

**Pull request merged**